server/server.go

```diff
@@ -188,6 +192,22 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 		m.blockchain.SetConsensus(m.consensus)
 	}
 
+	if m.config.Tracker {
+		//	create and start event tracker
+		m.tracker, err = tracker.NewEventTracker(
+			logger,
+			6, /* default */
```

Comment:
What do you think about extracting this value to a constant we can easily modify later?

By itself, it's just a magic number 🙂

Reply:
[Exported](https://github.com/0xPolygon/polygon-edge/pull/386/commits/47545725885115a68fa5d69b3d823bb11c6d96df)